data/korath/korath weapons.txt

```diff
@@ -995,8 +995,8 @@ outfit "Firelight Activated"
 		"optical tracking" .7
 		"trigger radius" 30
 		"blast radius" 60
-		"shield damage" 950
-		"hull damage" 950
+		"shield damage" 650
+		"hull damage" 650
```

Comment:
I'm not a maths person, but the Firelight and Firestorm are *supposed* to be ahead of the curve of other korath weapons. They represent a sort of weapons technology transition, the appearance of the (as yet unimplemented) `nra`ret meant to mirror real-life late 20th century migration away from battleships and towards missile cruisers.